### PR TITLE
Add reqbin-zip extension

### PR DESCRIPTION
**Pull request by rupert648**
## Description
A pretty simple extension to integrate with my side project [ReqBin.zip](https://reqbin.zip) - a developer friendly paste bin.

This extension adds two commands
- **Create New** : Gives a form which mimics the homepage of ReqBin.zip - i.e. a textarea to enter the text to be in the pastebin, plus some additional options.
- **Copy Contents** : Given a link to a reqbin, pulls the contents from that reqbin and copies it to the clipboard.

Let me know if anythings missing/needs changing!

## Screencast

Quick demo of the two commands:
https://github.com/raycast/extensions/assets/47698091/fabd0b10-5af4-4434-8879-c81231b5018b

## Checklist
- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/reqbin-zip/src/constants/url.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const url = process.env.NODE_ENV === "development" ? "http://localhost:3000" : "https://reqbin.zip";
```
Comment (mil3na) [inline extensions/reqbin-zip/src/constants/url.ts:1]: Other than you, would any user benefit from having localhost here? It seems a bit odd having it in prod.

**Review by mil3na — CHANGES_REQUESTED** [review]
Hi! Thank you so much for your contribution to Raycast!
My name is Milena and I will be reviewing your extension today :)

I'm trying to create a reqBin with some Loren Ipsum - as a regular dev does - and I'm getting both the error below and it failing silently:
CleanShot 2023-07-14 at 14 30 41@2x

CleanShot 2023-07-14 at 14 30 12@2x

It works on your website tho: https://www.reqbin.zip/clk2k6p060000mg08ccdh4mfw

However, entering this same URL on the `Copy Contents` command gives me the following error.
CleanShot 2023-07-14 at 14 32 27@2x

Could you please take a look?

**Comment by mil3na** [comment]
Hi! Any updates on my comments? Anything I can help you with?

**Comment by rupert648** [comment]
> Hi! Any updates on my comments? Anything I can help you with?

Hi! I unfortunately haven't had any time recently to revisit this, but thank you for your review. I'll close the PR for now to avoid clutter and will return to it at some point if I find time.

**Pull request closed without merging**